newIDE/app/src/Profile/EmailVerificationDialog.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import React from 'react';
+import FlatButton from '../UI/FlatButton';
+import Dialog, { DialogPrimaryButton } from '../UI/Dialog';
+import { type AuthenticatedUser } from './AuthenticatedUserContext';
+import BackgroundText from '../UI/BackgroundText';
+import VerifiedUser from '@material-ui/icons/VerifiedUser';
```

Comment:
You could use this icon from our new icons:
<img width="72" alt="image" src="https://user-images.githubusercontent.com/32449369/216923365-5e145ea2-a808-4bb5-a081-6e5e5b12ca92.png">

Reply:
I'll use this one